BE/toDoListAPI/src/main/java/shcool/mindera/toDoListAPI/controller/TaskController.java

```diff
@@ -0,0 +1,42 @@
+package shcool.mindera.toDoListAPI.controller;
+
+import org.springframework.web.bind.annotation.*;
+import shcool.mindera.toDoListAPI.model.DTOGroup;
+import shcool.mindera.toDoListAPI.model.DTONewComment;
+import shcool.mindera.toDoListAPI.model.DTONewTask;
+import shcool.mindera.toDoListAPI.model.DTOTask;
+import shcool.mindera.toDoListAPI.service.TaskService;
+
+import javax.swing.plaf.PanelUI;
+import java.util.List;
+
+@RestController
+@RequestMapping("todo/tasks")
+public class TaskController {
+    TaskService taskService;
+
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    //tasks
+    @GetMapping("/{userId}")
+    public List<DTOTask> getTasks(@PathVariable Integer userId){
+        return null;
+    }
+    @PostMapping("/new-task")
+    public void createTask(@RequestBody DTONewTask newTask){
+        //save Task
+    }
+
+    //comments
+    @GetMapping("/comments")
+    public List<DTOGroup> getGroups(@PathVariable Integer taskId){
+        return null;
+    }
+    @PostMapping("/comments/new")
+    public void createComment(@RequestBody DTONewComment newComment){
+        //save comment
+    }
+
```

Comment:
Comments dentro da task? Nao acho necessario

Reply:
Mas os comments não vão estar sempre relacionados às tasks? Onde seria melhor por?

Reply:
Dentro do seu proprio controller. Mesmo se tiveres dentro da task continuas a conseguir acessar o endpoint do comment, sem falar que em termos organizaçao nao é fixe se for num projeto grande com 500 tabelas como vais encontrar uma especifica vai ser muito dificil, vais ter milhares de endpoints na mesma class.

Reply:
faz sentido, em questão de organização fica melhor separar visto que quanto mais isso cresce mais difícil fica dar manutenção